src/resources/faqs.yml

```diff
@@ -1,57 +1,51 @@
 
   - question: Does this cost money?
-    answer: No, entry is completely free. In fact, we will even be able reimburse some of your travel costs.
+    answer: Entry is completely free — in fact, we'll even reimburse some of your travel costs. However, if you are arriving internationally, we won't be able to cover a significant portion of your costs.
 
   - question: Why do I need to provide a CV?
     answer: Select sponsors will have access to your CV. This is so that you can get on their radar and they can get access to you for recruitment purposes. Win-win!
 
   - question: What will I eat?
-    answer: |
-      Free food (and coffee) will be available at the event.
-
-      You won’t go hungry, trust us.
+    answer: We'll provide the food and drinks, leaving you to concentrate on your project.
 
   - question: Where will I sleep?
-    answer: The best solution to this is prepare. Sleep well beforehand and feel free bring a sleeping bag if you wish to sleep during the hackathon. If you a travelling a long distance contact us after your application and we will try our best to accommodate you.
+    answer: We'll have a room to nap in during the event — so bring a sleeping bag just in case. If you're coming from farther away, we're working on a Host-a-Hacker scheme to provide to crash on the Friday evening — more information on that coming soon.
 
   - question: What should I bring?
-    answer: A laptop is a big must, anything else depends on what you want to do at the hackathon. Bring along any hardware that you might want to hack on.
+    answer: You'll need a laptop for your hacking — everything else depends on your project. Bring along any hardware you might need.
 
   - question: What if I don’t know what I want to do?
-    answer: We will be providing experienced people and ideas to help you with. We’ll also suggest ideas and themes leading up to the hackathon, as well as during the event.
+    answer: Joining a team is a great way to develop ideas with others, and to make sure you're making the most of the hackathon. When you apply, you can ask us to place you in a team so you can hit the ground running.
 
   - question: How do I get there?
-    answer: The nearest station to Hack Cambridge is the Cambridge train station. From there it’s a 20 minute walk to the Corn Exchange.
+    answer: The nearest station to Hack Cambridge is the Cambridge train station. From there it’s just a 20 minute walk to the Corn Exchange.
 
   - question: How do teams work?
-    answer: We support teams of up to 4 members. If you want to apply as a group, first submit your individual application. Once all members of your planned team have submitted their individual applications, one you can complete [this team form](/teamapply).
+    answer: You can have teams of up to 4 members. After you submit your application, you'll be able to form a team [from the dashboard](/apply/dashboard).
```

Comment:
I think we should clarify that these teams are simply for the application process